_summaries/zeiler2012adadelta_1.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{zeiler2012adadelta}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author:  sritank
+score:  10
 ---
 
 TODO: Summarize the paper:
+
 * What is the core idea?
+- Adaptively change the learning rate in a way that it reduces oscillations i.e. slows down close to minimas, and is large otherwise.
+- Change step size based on the windowed accumulated gradient and also accumulated step sizes, which only require 1st order computations.
```

Comment:
may not need to add but would be good to mention that this windowed accumulation also makes sure that updates after many rounds of training are impactful and not "drowned out" by the massive accumulated denominator from ALL previous steps (as in ADAGRAD)

Reply:
Added it